src/dials/command_line/generate_mask.py

```diff
@@ -15,8 +17,11 @@
     untrusted.circle=200,200,100
 
   dials.generate_mask models.expt d_max=2.00
-"""
 
+  dials.generate_mask models.expt d_max=2.00 border.mask
+
+  dials.generate_mask beamcenter.mask border.mask
```

Comment:
Suggest for most of our users `backstop.mask` and `shadow.mask` may be more useful example names